srcpkgs/libco/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'libco'
+pkgname=libco
+version=20
+revision=1
+build_style=gnu-makefile
+short_desc="Cooperative multithreading library written in C89"
+maintainer="Julio Galvan <juliogalvan@protonmail.com>"
+license="ISC"
+homepage="https://github.com/canonical/libco"
```

Comment:
IMO, this should be the homepage https://code.byuu.org/libco here but keep the distfiles to make it far simpler.

Reply:
There are changes to the github repo that have not been upstreamed and will likely never be upstreamed. So I would rather keep the homepage as-is.